### PR TITLE
[emby] Initial contribution

### PR DESCRIPTION
**Pull request by volfan6415**
This request picks up on the initial contribution from [[WIP][emby] Initial Contribution by volfan6415 · Pull Request #7575 · openhab/openhab-addons (github.com)](https://github.com/openhab/openhab-addons/pull/7575) but updates the binding for support in openhab3 as well as other code clean up and improvements.

This binding interfaces openHAB with a running instance of EMBY Media Server (https://emby.media/). EMBY is a PLEX alternative and the binding provides many of the same features for users of EMBY as the PLEX binding does for uses of PLEX.
I have used the version of the binding which was previously submitted for over two years now (albeit in openHAB 2.X as the binding was not compatible until now with openHAB 3)(and as referenced in this thread [[New Binding] EMBY Server Binding - Alpha - Add-ons / Bindings - openHAB Community](https://community.openhab.org/t/new-binding-emby-server-binding-alpha/54010/94) I believe others in the community have been using it for awhile now as well. It has worked well for the users that have tested it for me.

I have made the necessary updates to make the binding operate in openHAB 3 and now feel that this binding is ready to be incorporated. The core works well.  While there are certainly features that can be added over time the binding in its current state more than covers the basics of what someone using openHAB would want to get from EMBY.  It allows for interaction with the EMBY server to get currently playing devices/media which can then be used inside of openHAB as triggers for automations and the like.
I would like to flag that there will need to be clarification on is the author signing. In developing this binding I started with the KODI binding code, as EMBY provides a similar websocket api. While I did make extensive changes to work with the API provided by EMBY, there is definitively a structure left from the KODI binding and there are some methods and classes where the only real change was method/class names. Unfortunately in making these changes did not keep the author tags on the classes and now have all of the classes showing me as the original contribution.  It is not my intention to not give credit where credit is due, and so will need some guidance on what edits need to be made to the author tags in the file to rectify this situation.

I know there are significant updates which need to be made to the readme and I will be working on those put wanted to start this pull request so i could get some comments and suggestions on the underlying code before i spend too much time on the readme as that may very well change.

### CODE REVIEW

**Comment by lsiepel** [comment]
If you can fix the merge conflict, i can do a first review.

File: bundles/org.openhab.binding.emby/pom.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>3.4.0-SNAPSHOT</version>
```
Comment (wborn) [inline bundles/org.openhab.binding.emby/pom.xml:10]:
```suggestion
    <version>4.0.0-SNAPSHOT</version>
```

**Comment by volfan6415** [comment]
@lsiepel i am not sure exactly what the best way to resolve the merge conflict in the pom file is.  I thought the only thing i had done to that file was to add the emby binding into it but clearly not.  

Any suggestions on the best way to resolve would be helpful.

File: bundles/org.openhab.binding.emby/src/main/resources/OH-INF/binding/binding.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (lsiepel) [inline bundles/org.openhab.binding.emby/src/main/resources/OH-INF/binding/binding.xml:1]: This needs to be adapted to the new `addon.xml`, the PR to adapt the developer docs is not merged yet, so if you need more details, the corresponding issue is here: https://github.com/openhab/openhab-core/issues/2058
But you can also update your branch and look at another binding's `addon.xml` and adapt it to this initial contribution.

**Review by lsiepel — CHANGES_REQUESTED** [review]
Partial review.
There are some SAT errors and i think the log levels need to be adjusted. See: https://www.openhab.org/docs/developer/guidelines.html#f-logging

File: bundles/org.openhab.binding.emby/pom.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>4.0.0-SNAPSHOT</version>
```
Comment (wborn) [inline bundles/org.openhab.binding.emby/pom.xml:10]:
```suggestion
    <version>4.1.0-SNAPSHOT</version>
```

**Comment by lsiepel** [comment]
ping @volfan6415 do you plan to finish this PR?

**Comment by lsiepel** [comment]
Does not look like this PR will get to the finish line. @volfan6415 please re-open (or comment) on this PR if you want to continue. Due to no responses i close this PR for now.

**Pull request closed without merging**